handbook/data-stores/data-stores.md

```diff
@@ -0,0 +1,89 @@
+# MongoDB
+
+MongoDB is used by the majority of SystemLink services as the primary datastore. MongoDB Community Edition is included in the default installation of SystemLink will launch when the SystemLink server boots.
+
+!!!note
+    By default SystemLink stores all data on databases and the file system hosted on the SystemLink application server. This is referred to as a *single node* deployment. To improve the reliability of data storage and scale to larger volumes of data, remote data stores are recommended for all production deployments. This is referred to as a *multi node* deployment.
+
+## Assumptions and Prerequisites
+
+- A server running SystemLink 2021R1 or greater. Refer to Installing and Configuring SystemLink Server and Clients for the basics of setting up a SystemLink server.
+- A standalone server running MongoDB, multiple servers hosting a MongoDB replica set, or a MongoDB Atlas Account with a user with the `dbAdmin` or similar role. Refer to [Role-Based Access Control](https://docs.mongodb.com/manual/core/authorization/) for details.
+
+### SystemLink with Single Node vs Multi Node MongoDB Deployments
+
+The following table summarizes when to consider the various supported configurations of MongoDB with SystemLink.
+
+| Diagram | SystemLink + MongoDB Configuration | When to use |
+| | ----------- | ----------- |
+|<img src="../../img/single-box.png" width="1500px"/>| Single node with Defaults     | Here SystemLink and MongoDB are hosted on the same server. Use this configuration when evaluating SystemLink or in small deployments of less than 10 managed nodes without the Asset Manager and Test Insights modules. |
+|<img src="../../img/single-box.png" width="1500px"/>| Single node with increase index cache   | Here SystemLink and MongoDB are hosted on the same server. Use this configuration when evaluating SystemLink or in moderate deployments of less than 25 managed nodes and moderate usage of the Test Insights module producing less than 100 test steps and results each day.     |
```

Comment:
>Single node with increase index cache |  Here SystemLink and MongoDB are hosted on the same server.

This use case confuses me. Introductory note implies that using MongoDB always means going the multi-node deployment route. Consider either adding some more context to that first note where we define single-node deployment or adding that here.

Reply:
So I decided to include a discussion of the index cache after seeing a recent post on Teams from Josh where he was recommending that and realizing we didn't already have this workflow in SPD. Its worthwhile to document for the folks who aren't in a position to move to a multi-node deployment.